components/31-molecules/opening-hours/_opening-hours.scss

```diff
@@ -130,10 +130,10 @@ $week-breakpoint: '(min-width: 750px)';
 
         .openinghours--month {
           @include bold-text;
-          line-height: $month-day-size-mobile / 2;
+          line-height: $month-day-size-mobile *.5;
 
           @include phablet {
-            line-height: $month-day-size / 2;
+            line-height: $month-day-size *.5;
```

Comment:
Add space between * and .5. Makes it clearer that it is a calculation instead of a wildcard.